src/dlgIRC.cpp

```diff
@@ -1,6 +1,8 @@
 /***************************************************************************
+ *   Copyright (C) 2008-2017 The Communi Project                           *
  *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
  *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2017 by Fae - itsthefae@gmail.com                       *
```

Comment:
I do live in the US.   I wouldn't disavow myself from code I've written, that just wouldn't be right.   
I plan to support what I add for as long as is needed, but I would rather do it through this handle.    
I have no desire to publish my name for various reasons.  In the future this could change but I don't see it happening any time soon. 
A contact email address and this Github account are honestly all I want to publish about myself and should be ample for contacting me should there be any need by anyone inclined.